RationaleMCP/0031/differences.md

```diff
@@ -50,6 +50,230 @@ The rule for `pure(impureFunctionCall(...))` needs to be rephrased to not say _a
 
 This change was made to support the [changed definitions of _constant expression_](#Constant-expressions).
 
+## Function default arguments
+
+Flat Modelica functions cannot have function default arguments.
+A tool producing Flat Modelica from full Modelica can accommodate this by automatically generating a helper function for every present subset of argument used in the model.
```

Comment:
```suggestion
A tool producing Flat Modelica from full Modelica can accommodate this by automatically generating a helper function for every present subset of arguments used in the model.
The helpers can be omitted if the defaults are literal or otherwise independent of the other inputs, since those values can be added in each call.
```
The reason for stating this is both to avoid unnecessary helpers, and to implicitly explain why they are needed in advanced cases (where the defaults depend on other inputs)